README.rst

```diff
@@ -50,14 +50,18 @@
    :target: https://colab.research.google.com/github/lanl/scico-data/blob/colab/notebooks/index.ipynb
    :alt: Run notebooks on google colab
 
+.. image:: https://joss.theoj.org/papers/10.21105/joss.04722/status.svg
+   :target: https://doi.org/10.21105/joss.04722
+   :alt: JOSS paper
+
 
 
 Scientific Computational Imaging COde (SCICO)
```

Comment:
I suggest changing this to `Scientific Computational Imaging Code (SCICO)` as in the paper

Reply:
Done.